MANIFEST.in

```diff
@@ -9,3 +9,6 @@ recursive-include tests *.py
 include LICENSE
 include docs/LICENSE_THIRD_PARTY
 include docs/source/license.rst
+include cupy/cuda/cupy_cufftXt.cu  # for cuFFT callback
```

Comment:
These files are needed in wheels, so please add to package_data instead.

Reply:
Hi @kmaehashi I already added them to https://github.com/cupy/cupy/blob/3822ee86d3ab40424415187789419ca4974876f8/setup.py#L111-L115, did you mean I don't need to touch MANIFEST.in?

Reply:
Ah, sorry I overlooked the change in `setup.py` :bow:  I'm unsure how conflicting restrictions between setup.py and MANIFEST.in (`recursive-exclude cupy *.pyx`) work...

Now I come up with another problem: sdist assumes that pyx files are not included so that Cython does not run when building from sdist (#920). https://github.com/cupy/cupy/blob/1374d2aee7e9183ef560832fec5dfe91e7391f3b/cupy_setup_build.py#L317-L325
Maybe it's better to rename the current `cufft.pyx` to `cufft.pyi` and use `include "cufft.pyi"` from pyx so that only pxi is included in sdist?

Reply:
Ouch, sounds a bit more complicated than I expected... @kmaehashi, is it possible to make a copy of `cupy/cuda/cufft.pyx` during the process of generating sdist? Maybe I need to touch `sdist_with_cython()` in `cupy_setup_build.py`? Renaming it to `cufft.pyi` will likely work, but is a bit intrusive and I prefer it to be our last resort.

Reply:
Copying it in `sdist_with_cython` sounds good to me.

Reply:
Hi @kmaehashi I looked into that, and realized the current `MANIFEST.in` is already enough to make the sdist include the two files `cufft.pxd` and `cufft.pyx` (checked by running `python setup.py sdist`). I discarded an unnecessary commit made yesterday. 

So to your two questions:
> I'm unsure how conflicting restrictions between setup.py and MANIFEST.in (recursive-exclude cupy *.pyx) work...

Looks like MANIFEST.in is executed *after* `package_data` is set (see https://packaging.python.org/guides/using-manifest-in/). So the two files will eventually be added back when building both sdist and wheel.

> Now I come up with another problem: sdist assumes that pyx files are not included so that Cython does not run when building from sdist

Given that the two files are present, when building from sdist:
1. If Cython is available, we'll do cythonize and then build
2. If Cython is unavailable, we will just build (skipping cythonize)

So, I think the present approach should just be sufficient to make the both files included and installed. Could you check if I'm making sense? 🙂

Reply:
@kmaehashi Any comments? 🙂